tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -109,7 +117,7 @@ def test_failed_vetext_call_throws_general_exception(mocker,  event):
 
 @pytest.mark.parametrize('event', [(sqsInvokedWithAddOn)])
 def test_failed_sqs_invocation_call_throws_general_exception(mocker, event):
```

Comment:
How is this function directly testing that HttpException was raised?

Reply:
if exception wasnt raised then retry sqs wouldnt be calle